components/Card.js

```diff
@@ -19,4 +19,15 @@ export default function Card(cardElement) {
     }
     answerText.classList.toggle('card__answer--hide');
   });
+
+  const apiUrl = 'https://jservice.io/api/random';
+  fetch(apiUrl)
+    .then(response => response.json())
+    .then(data => createQuizCard(data))
+    .catch(error => console.log(error));
+
+  function createQuizCard(card) {
+    questionText.innerHTML = `<p>${card[0].question}</p>`;
```

Comment:
I also want to use the procedure via the innerHTML method. At the moment I am still using textContent.

Reply:
textContent did not work for me, cause I am using a paragraph for the questions & answers